client/components/UserSettings/index.jsx

```diff
@@ -1,14 +1,86 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Checkbox from '@components/Checkbox';
+import { handleGetValidAts } from '../../actions/ats';
 
 class UserSettings extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {};
+
+        this.checkBoxClicked = this.checkBoxClicked.bind(this);
+    }
+
+    componentDidMount() {
+        const { dispatch } = this.props;
+        dispatch(handleGetValidAts());
+    }
+
+    componentDidUpdate(prevProps) {
+        if (prevProps.ats !== this.props.ats) {
+            this.setState(
+                Object.assign(
+                    {},
+                    ...this.props.ats
+                        .map(at => [at, { checked: false }])
+                        .map(([k, v]) => ({ [k]: v }))
+                )
+            );
+        }
+    }
+
+    checkBoxClicked(event) {
```

Comment:
Suggestion: It might be easier to associate this in the future if it's named closer to the functionality it provides. I just ran into this in TSL/TM. A convention that helps for me is "on" + "thing" + "event", ie. `onCheckboxClick`. This is obviously not a blocker, but like I said, I encountered some confusing event handler names in TSL/TM yesterday and ended up renaming them to match this pattern.